category_encoders/__init__.py

```diff
@@ -31,5 +32,6 @@
     'PolynomialEncoder',
     'BaseNEncoder',
     'LeaveOneOutEncoder',
-    'TargetEncoder'
+    'TargetEncoder',
+    'WoeEncoder'
```

Comment:
Should we capitalize WOE since it's an acronym? I'm not sure there is a right or wrong, but whichever we go with here we should remember to keep consistent in future classes.

Reply:
PEP 8 says WOE -> changed.